pkg/migration/templates/kubeadm_controlplane_azure.yaml.tmpl

```diff
@@ -0,0 +1,286 @@
+        [Install]
+        WantedBy=multi-user.target
+      owner: root:root
+      path: /etc/systemd/system/azure-cni-nat-rules.service
+      permissions: "0644"
+    - content: "whites ALL = (ALL) NOPASSWD: ALL"
+      owner: root:root
+      path: /etc/sudoers.d/whites
+      permissions: "0440"
+    - content: "tuommaki ALL = (ALL) NOPASSWD: ALL"
+      owner: root:root
+      path: /etc/sudoers.d/tuommaki
+      permissions: "0440"
+    - contentFrom:
+        secret:
+          key: tls.crt
+          name: {{.ClusterID}}-ca
+      path: /etc/kubernetes/pki/etcd/ca.crt
+      permissions: "0640"
+      owner: root:root
+    - contentFrom:
+        secret:
+          key: tls.key
+          name: {{.ClusterID}}-ca
+      owner: root:root
+      path: /etc/kubernetes/pki/etcd/ca.key
+      permissions: "0600"
+    - contentFrom:
+        secret:
+          key: tls.crt
+          name: {{.ClusterID}}-ca
+      owner: root:root
+      path: /etc/kubernetes/pki/ca.crt
+      permissions: "0640"
+    - contentFrom:
+        secret:
+          key: tls.key
+          name: {{.ClusterID}}-ca
+      owner: root:root
+      path: /etc/kubernetes/pki/ca.key
+      permissions: "0600"
+    - contentFrom:
+        secret:
+          name: {{.ClusterID}}-etcd
+          key: "tls.crt"
+      owner: root:root
+      path: /etc/kubernetes/pki/etcd/old-etcd-cert.pem
+      permissions: "0640"
+    - contentFrom:
+        secret:
+          name: {{.ClusterID}}-etcd
+          key: "tls.key"
+      owner: root:root
+      path: /etc/kubernetes/pki/etcd/old-etcd-key.pem
+      permissions: "0640"
+    - contentFrom:
+        secret:
+          name: {{.ClusterID}}-service-account
+          key: "tls.crt"
+      owner: root:root
+      path: /etc/kubernetes/pki/sa.pub
+      permissions: "0640"
+    - contentFrom:
+        secret:
+          name: {{.ClusterID}}-service-account
+          key: "tls.key"
+      owner: root:root
+      path: /etc/kubernetes/pki/sa.key
+      permissions: "0640"
+    - contentFrom:
+        secret:
+          name: {{.ClusterID}}-k8s-encryption-config
+          key: encryption
+      owner: root:root
+      path: "/etc/kubernetes/encryption/k8s-encryption-config.yaml"
+      permissions: "0644"
+    - contentFrom:
+        secret:
+          name: {{.ClusterID}}-proxy-config
+          key: proxy
+      owner: root:root
+      path: "/etc/kubernetes/config/proxy-config.yml"
+      permissions: "0644"
+    - contentFrom:
+        secret:
+          name: {{.ClusterID}}-kubeconfig
+          key: value
+      owner: root:root
+      path: "/etc/kubernetes/config/proxy-kubeconfig.yaml"
+      permissions: "0644"
+    - content: |
+        #!/bin/sh
+        # create etcd ca bundle
+        cat /etc/kubernetes/pki/etcd/ca.crt /etc/kubernetes/pki/etcd/old-etcd-ca.pem > /etc/kubernetes/pki/etcd/ca-bundle.pem
```

Comment:
Where does old-etcd-ca.pem come from?

Reply:
Ping @whites11 here as well.

Reply:
This part comes from Vaclav's work. Anyway the old CA comes from the etcd secret we have in MCs (created by cert-operator)

Reply:
I don't see where it is mounted. Maybe I'm missing something.

Reply:
you are right it doesn't seem to be mounted anywhere.
The bundle is not used anywhere also, so I think we can get rid of this line

Reply:
So we don't use the bundle? Or `/etc/kubernetes/pki/etcd/ca.crt` is already the bundle?